nuxeo-web-ui-ftest/webdriver/test/features/group.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Group
+
+  As an Administrator I can create, edit search for and delete groups.
+
+  Background:
+    Given I login as "Administrator"
+    And I click the "administration" button
+    And I can see the administration menu
+    And I click "Users & Groups" in the administration menu
+    And I can see the users and groups page
```

Comment:
It was in case we added more scenarios in the future, but ok. Although if we are separating Create, Edit and Delete into different scenarios (see below) then I suppose we should keep it as a background.